src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -83,7 +95,7 @@ abstract contract DelegationManagerStorage is IDelegationManager {
      * To withdraw from a strategy, max(minWithdrawalDelayBlocks, strategyWithdrawalDelayBlocks[strategy]) number of blocks must have passed.
      * See mapping strategyWithdrawalDelayBlocks below for per-strategy withdrawal delays.
      */
-    uint256 public minWithdrawalDelayBlocks;
+    uint256 private __deprecated_minWithdrawalDelayBlocks;
```

Comment:
We should keep using timestamps as it doesn't seem there's a good reason to switch to blocks. This would just be another moving part in an already huge release so whatever we can keep the same, we should.

Reply:
Typo? Do you mean we should keep using blocks vs switching to timestamps